pkg/alertmanager/alertmanager_test.go

```diff
@@ -778,17 +779,43 @@ func TestGetFullStateHandler(t *testing.T) {
 		rec := httptest.NewRecorder()
 		am.GetFullStateHandler(rec, nil)
 		require.Equal(t, http.StatusOK, rec.Code)
+		require.Equal(t, "application/json", rec.Header().Get("Content-Type"))
+
 		body, err := io.ReadAll(rec.Body)
 		require.NoError(t, err)
-		json := `
-		{
-			"data": {
-				"state": "CgoKCG5mbDp0ZXN0CgoKCHNpbDp0ZXN0"
-			},
-			"status": "success"
+		var (
+			parsedBody struct {
+				Status string `json:"status"`
+				Data   struct {
+					State string `json:"state"`
+				} `json:"data"`
+			}
+			parsedState clusterpb.FullState
+		)
```

Comment:
This is a bit hard to read.
Wdyt about declaring just `parsedBody` here?
```suggestion
		var parsedBody struct {
			Status string `json:"status"`
			Data   struct {
				State string `json:"state"`
			} `json:"data"`
		}
```

We don't need `parsedState` until line 800.

Reply:
Suure, done now 👍